### ISSUE TITLE
[UX] Create List of Questions for Dr. Hsu

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#38859]
## Problem Statement
We are being given the opportunity to interview Dr. Warren Hsu for telephone appointments and we need to draft a list of questions for him.

## Acceptance Criteria
- [ ] Create list of questions (repurpose questions asked of Shane)

**Comment by benbrasso-agile6** [comment]
@kristenmcconnell can you take a look at the questions I put together? https://docs.google.com/document/d/1aEO9Y-e4pFOoghtCTuakqLAWDGE08fjhW-7EZkROjOE/edit?usp=sharing

These are repurposing the questions asked to Shane with some modifications.

**Comment by benbrasso-agile6** [comment]
Let me know if and/or where we should drop this into Git. I'm happy to do so.

**Comment by kristenmcconnell** [comment]
@benbrasso-agile6 I added a few comments to the document. Overall, it looks really good and I believe the number of questions will work well for the 30 minutes with have with Dr. Hsu. 

Definitely go ahead and post this to GitHub. I think we could add it within one of two locations.
1. Nira and I created this [GitHub folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/discovery/telephone-appointments) with all of our telemedicine discovery items. 
2. Somewhere in the [staff-facing research folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/staff-facing).
